tests/test_workload.py

```diff
@@ -1,13 +1,22 @@
 from unittest.mock import patch
-from uuid import uuid4
+from uuid import uuid4, UUID
 
 import pytest
+from pydantic import ValidationError
 
 from pasqal_cloud import SDK, Workload
 from tests.test_doubles.authentication import FakeAuth0AuthenticationSuccess
 
 
 class TestWorkload:
+    @pytest.fixture
+    def workload_with_link_id(self):
+        return UUID(int=0x2)
```

Comment:
Might be better to convert it to string in case of it is used for other tests later on that requires it as a string